### PR TITLE
Arm subsystem

### PR DESCRIPTION
**Pull request by admlvntv**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/arm/ArmIOSparkMAX.java
Comment (Aradig26) [inline src/main/java/frc/robot/subsystems/arm/ArmIOSparkMAX.java:1]: Move motor ids to constants file

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -90,6 +94,7 @@ public RobotContainer() {
                   new ModuleIOSparkMAX(1),
                   new ModuleIOSparkMAX(2),
                   new ModuleIOSparkMAX(3));
+          arm = new Arm(new ArmIOSparkMAX());
```
Comment (EQMoore) [inline src/main/java/frc/robot/RobotContainer.java:97]: Remove this

**Review by EQMoore — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/arm/ArmConstants.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+public class ArmConstants {
+  public static final TunableNumber KG;
+  public static final TunableNumber KS;
+  public static final TunableNumber KV;
+  public static final TunableNumber KP;
+  public static final TunableNumber KD;
+  public static final TunableNumber MAX_VELOCITY;
+  public static final TunableNumber MAX_ACCELERATION;
+  public static final Rotation2d
+      horizontalOffset; // position between our 0 (neutral pick up position) and real 0 (parallel to
```
Comment (EQMoore) [inline src/main/java/frc/robot/subsystems/arm/ArmConstants.java:17]: this should be HORIZONTAL_OFFSET

**Review by EQMoore — CHANGES_REQUESTED** [review]

**Review by EQMoore — APPROVED** [review]
LGTM

**Pull request merged**